prow/cmd/sinker/main.go

```diff
@@ -239,9 +265,20 @@ func (c *controller) clean() {
 			// Delete old finished or orphan pods. Don't quit if we fail to delete one.
 			if err := client.Delete(pod.ObjectMeta.Name, &metav1.DeleteOptions{}); err == nil {
 				c.logger.WithField("pod", pod.ObjectMeta.Name).Info("Deleted old completed pod.")
+				c.currentMetrics.podsTotalRemovedByProw++
+				// TODO(hongkliu) replace this with the real reason
+				c.currentMetrics.addPodByReason("unknown", c.logger)
```

Comment:
Maybe both numbers should be bumped inside the method?

Reply:
podsTotalRemovedByProw is removed now.
Will be calculated on the fly as @stevekuznetsov pointed out.